echo/test/unit/echo_client_test.js

```diff
@@ -237,6 +237,75 @@ describe("Echo Node Client Test Suite", function(){
                 result.body.props.should.be.an.object;
                 result.body.props.url.should.equal("https://foo.bar/baz.html");
 
+                done();
+            });
+        });
+        it("- rounds the timestamp to nearest whole number for arrays of events", function(done){
+            var echoClient = echo.createClient({
+                echo_endpoint: endPoint
+            });
+            
+            var requestStub = sandbox.stub(request, 'post')
+            requestStub.callsFake(function (options, callback) {
+                callback(
+                    null,
+                    {statusCode: 200},
+                    {
+                        "class": "page.views",
+                        "timestamp": 1324524509,
+                        "user": "1234-5678-9012-3456",
+                        "source" : "elevate-app",
+                        "props": {
+                            "url" : "https://foo.bar/baz.html"
+                        }
+                    }
+                );
+            });
+
+            const events = [{class:'class', source:'source', timestamp: 1324524508 }, {class:'class', source:'source', timestamp: 1324524509.123 }];
+            
+            echoClient.addEvents('secret', events, function(err){
+                const updatedEvents = [{class:'class', source:'source', timestamp: 1324524508 }, {class:'class', source:'source', timestamp: 1324524509 }];
+                
+                (err === null).should.be.true;
```

Comment:
```suggestion
                (err === null).should.be.true();
```

Reply:
I copied this from the other tests and I think it's ok?
https://github.com/tj/should.js/#true

Reply:
This looks like old documentation to me?

http://shouldjs.github.io/#assertion-true

As far as I understand it, `should` assertions need to be methods. Hence installing a linter in Elevate to ensure this happens.

Reply:
I agree there is an inconsistency with their readme and their docs site. I have no preference either way so I was follow the 'house style'. So the question is do we want to add in consistency here? What is the reason for preferring the method call?

Reply:
Change this line to `(err === null).should.be.false`.

It will pass.

This is incorrect.

Reply:
In order to ensure the statement is asserted, you must call the method (so `true()` or `false()`), otherwise this is effectively a noop.